revolv/base/forms.py

```diff
@@ -21,6 +21,7 @@ def save(self, commit=True):
         last names.
         """
         user = super(SignupForm, self).save(commit=False)
+        user.save()
```

Comment:
Why add this? User.save() is not supposed to happen unless `commit` is true, which is down on line 29